src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -81,6 +81,15 @@ public void Configure(TelemetryConfiguration configuration)
             {
                 module.Initialize(configuration);
             }
+
+            foreach (ITelemetryProcessor processor in configuration.TelemetryProcessors)
+            {
+                ITelemetryModule module = processor as ITelemetryModule;
```

Comment:
So I udnerstand the situation where a class implements both ITelemetryProcessor and ITelemetryModule but why wouldn't the ITelemetryModule also be registered so that it is handled by the existing foreach above?